contrib/ubase/Makefile

```diff
@@ -0,0 +1,135 @@
+TOPDIR = $(realpath ../..)
+
+
+HDR = \
+	arg.h \
+	compat.h \
+	crypt.h \
+	fs.h \
+	md5.h \
+	queue.h \
+	sha1.h \
+	sha224.h \
+	sha256.h \
+	sha384.h \
+	sha512.h \
+	sha512-224.h \
+	sha512-256.h \
+	text.h \
+	utf.h \
+	util.h
+
+LIBUTILSRC = \
+	libutil/agetline.c       \
+	libutil/concat.c         \
+	libutil/ealloc.c         \
+	libutil/eprintf.c        \
+	libutil/estrtol.c        \
+	libutil/estrtoul.c       \
+	libutil/explicit_bzero.c \
+	libutil/passwd.c         \
+	libutil/proc.c           \
+	libutil/putword.c        \
+	libutil/recurse.c        \
+	libutil/strlcat.c        \
+	libutil/strlcpy.c        \
+	libutil/strtonum.c       \
+	libutil/tty.c
+
+LIBUTILSRC_DONT_WORK = \
+	libutil/agetcwd.c \
```

Comment:
Is there any reason why you skipped those?

Reply:
`agetcwd()` function uses `apathmax()`, which needs `pathconf()`. This is a syscall (https://man.netbsd.org/pathconf.2), that we don't have implemented.

Reply:
The system call should be relatively easy to add. Have a look at [sys_pathconf](http://bxr.su/NetBSD/sys/kern/vfs_syscalls.c#sys_pathconf) in NetBSD. Through redirection layers it reaches filesystem specific `VOP_PATHCONF` for instance [ufs_pathconf](http://bxr.su/NetBSD/sys/ufs/ufs/ufs_vnops.c#ufs_pathconf). Let me know if you'd like to implement it.

Reply:
Sounds great! Please create a GitHub issue for `pathconf` syscall and describe there how you'd approach the implementation.

Reply:
I've added the issue here: https://github.com/cahirwpz/mimiker/issues/1397